components/task/details/solutions/newSolutionForm/NewSolutionForm.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FormEventHandler, useState, ChangeEvent } from 'react';
+import Button, { ButtonType } from '../../../../shared/button/Button';
+import Select, { ValueType } from 'react-select';
+import { formStyles } from './formStyles';
+
+export const technologies = [
+  { value: 'html', label: '#html' },
+  { value: 'css', label: '#css' },
+  { value: 'js', label: '#js' },
+  { value: 'nodejs', label: '#nodejs' },
+  { value: 'python', label: '#python' },
+  { value: 'react', label: '#react' },
+  { value: 'vue', label: '#vue' },
+  { value: 'angular', label: '#angular' },
+];
+
+interface Technology {
+  value: string;
+  label: string;
+}
+
+const NewSolutionForm: React.FC = () => {
+  const [solutionLink, setSolutionLink] = useState('');
+  const [liveLink, setLiveLink] = useState('');
+  const [selectedTechnologies, setSelectedTechnologies] = useState<Technology[]>([]);
+
+  const handleSolutionChange = (e: ChangeEvent<HTMLInputElement>): void => {
+    setSolutionLink(e.target.value);
+  };
+
+  const handleLiveLinkChange = (e: ChangeEvent<HTMLInputElement>): void => {
+    setLiveLink(e.target.value);
+  };
+
+  const handleSelectedTechnologies = (selectedOpion: ValueType<Technology>): void => {
+    const selectedTechs = selectedOpion as Technology[];
+    setSelectedTechnologies(selectedTechs);
+  };
+
+  const handleSubmit: FormEventHandler = (e) => {
+    e.preventDefault();
+    console.log(selectedTechnologies);
+  };
+  return (
+    <form onSubmit={handleSubmit} action="" className="w-full flex flex-col items-start">
+      <div className="w-full lg:w-3/4">
+        <div className="flex flex-col pt-4 w-full">
```

Comment:
Jak dla mnie to powinien być Component

Reply:
Nie miałem za bardzo pomysłu na to wydzielenie. Zrobiłem `NewSolutionFormInput`, ale nie wiem czy zrobiłem to poprawnie.